### DISCUSSION TITLE
[BUG] uploadListingVideo call: The listing already has the maximum number of videos allowed.

### DISCUSSION THREAD

**Discussion by valigara** [etsy/open-api#682] [Q&A] [answered]
Despite fact that listing has no videos:
#### Request:
```
GET /v3/application/listings/1237742289/videos HTTP/1.1
Host: openapi.etsy.com
Accept: application/json
x-api-key: ***
authorization: Bearer ***
```
#### Response:
```
HTTP/1.1 200 OK
X-Etsy-Request-Uuid: EuAPJyIMvAcC3S1FD9YsNXfa9a7c

{"count":0,"results":[]}
```
While trying to add video to listing:
#### Request:
```
POST /v3/application/shops/6897505/listings/1237742289/videos HTTP/1.1
Transfer-Encoding: chunked
Expect: 100-Continue
Content-Type: multipart/form-data; boundary=05a8345695df491867b23f87bcced2edf30554d9
Host: openapi.etsy.com
Accept: application/json
x-api-key: ***
authorization: Bearer ***

***video file body removed***
```
#### Response:
```
HTTP/1.1 400 Bad Request
Connection: keep-alive
Content-Length: 74
Server: Apache
X-CLOUD-TRACE-CONTEXT: 79ac210ec169b49a21938f02ba986055/14014988084244265776;o=0
X-Etsy-Request-Uuid: EuuQ8juSEkj6r1fGhYM32FTwDW08
X-Limit-Per-Second: 20
X-Remaining-This-Second: 19
X-Limit-Per-Day: 60000
X-Remaining-Today: 59667
Content-Type: application/json
Via: 1.1 google, 1.1 varnish
Accept-Ranges: bytes
Date: Wed, 30 Nov 2022 09:28:46 GMT
X-Served-By: cache-mrs10543-MRS
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1669800479.845935,VS0,VE47921

{"error":"The listing already has the maximum number of videos allowed."}
```
etsy api returns non relevant error `The listing already has the maximum number of videos allowed.`

# Please advise!

p.s.
looks like work of `ShopListing Video` api calls is not stable, issue #672 that we mentioned earlier still reproduces.

**Comment by etsyachristensen** [comment]
@valigara In looking at that listing, there's already 10 videos attached to that listing, but all are marked as deleted. We will look into fixing this. Apologies for the issue.

**Comment by etsyachristensen** [comment, marked as answer]
On further investigation. There's a limitation built into the system. It prevents users from uploading more than 10 videos per 24 hour period per listing, regardless of whether they have been deleted. Looking at that listing, there were 10 videos uploaded within a 24 hour period. Our assumption is that the 11th was attempted before the 24 hour clock had run out.

We will have to get this into the documentation.
